test/cmd/newapp.sh

```diff
@@ -5,19 +5,25 @@ trap os::test::junit::reconcile_output EXIT
 # Cleanup cluster resources created by this test
 (
   set +e
-#  oc delete all,templates --all
-  oc delete-project template-substitute
-  oc delete-project prefix-template-substitute
+  oc login -u system:admin
+  oc delete all,templates --all
+  oc delete all,templates --all -n openshift
```

Comment:
this is potentially destructive to other tests using this cluster.

Reply:
Agreed, I also find it a bit odd to make use of the _openshift_ namespace here. However, this is not something I introduced, but this is already happening in the test see https://github.com/openshift/origin/blob/master/test/cmd/newapp.sh#L21 and https://github.com/openshift/origin/blob/master/test/cmd/newapp.sh#L57. Initially, the test run would fail pretty much immediately since I had templates imported in the _openshift_ namespace. This change makes sure that the right preconditions for this test suite are met. It might actually make more sense to use a dedicated namespace, but I am not sure whether there was a deeper thought behind choosing _openshift_ in the first place. 
FAICT, this does not impact the rest of the integration tests atm.

Should I change this to use another namespace?

Reply:
I think the reason for using the openshift namespace is that is where new-app looks for imagestreamtags, in addition to looking in the current namespace.

i think you can leave this.

Reply:
ok